adcircpy/mesh/base.py

```diff
@@ -17,8 +17,7 @@
 from matplotlib.tri import Triangulation
 import numpy as np
 from pyproj import CRS, Transformer
-from shapely.geometry import LineString, LinearRing, MultiPolygon, \
-    Point, Polygon, box
+from shapely.geometry import LineString, LinearRing, MultiPolygon, Point, Polygon, box
```

Comment:
this can be also done automagically with `isort`

Reply:
added in 8d9db1f